Code/Scripts/CMakeLists.txt

```diff
@@ -110,17 +112,29 @@ if(SV_USE_TKCXIMAGE)
 	endif()
 endif()
 
+if (SV_USE_SYSTEM_TCL)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+endif()
+
 if(NOT SV_USE_SYSTEM_TCL OR SV_INSTALL_SYSTEM_LIBS)
 	#Lib additions
-	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.5" developer_script_string)
-	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.5" developer_script_string)
-	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.5" install_script_string)
-	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.5" install_script_string)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.6" install_script_string)
+	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.6" install_script_string)
 endif()
 
 #--------
 # Python
 if(SV_USE_PYTHON)
+  if (SV_USE_SYSTEM_PYTHON)
+	   append_env_string_concat(PYTHONPATH "${PYTHON_SITE_PACKAGES}" developer_script_string)
+	if (SV_USE_SYSTEM_VTK)
```

Comment:
I had to explicitly set the VTK paths because I think the VTK_DIR and assumed directory structure are only correct if you use the VTK build directory and not the VTK installed directory (i.e. what happens when you are in SV16).  I like being explicit for this one.  I don't think the developer script was correct before anyway.  It explicitly added the path ".../site-packages/vtk" which I think you don't need since I think the packages works by searching ".../site-packages" looking for the directory vtk.   I guess it's possible there is a platform related issue with the structure related to where the python libs end up in the VTK_DIR structure, but I just assumed it was a difference between build/install structure.

In general, I think we should force the user to specify VTK_PYTHON_SITE_PACKAGES if SV_USE_VTK and SV_USE_PYTHON are specified.